src/net.md

```diff
@@ -831,6 +832,57 @@ shows Listening on 127.0.0.1:11500, run
 
 After sending data in telnet press `ctrl-]` and type `quit`.
 
+[ex-extract-links-webpage]: #ex-extract-links-webpage
+<a name="ex-extract-links-webpage"/>
+## Extract all links from a webpage
+
+[![reqwest-badge]][reqwest] [![select-badge]][select] [![cat-net-badge]][cat-net]
+
+```rust
```

Comment:
Please change 
<pre>```rust</pre>
to
<pre>```rust,no_run</pre>

We do not want to make network requests in our CI environment